### PR TITLE
Cindy schedule

### PR DESCRIPTION
**Pull request by cindy1u0**
This PR includes a basic calendar UI for schedule. It can read in data from a JSON file now but later should be changed to airTable. What it _**DOESN'T**_ have:
1. Gray out when it's happening now 
2. Tooltip for "happing now" and "add to calendar"
3. Mobile styling
4. Expanded styling (because I am still not sure what the design should look like...)
I can also keep working on this, but just want to put up a PR before this gets huge :DD
Screen Shot 2021-01-07 at 11 24 24 PM
Screen Shot 2021-01-07 at 11 24 19 PM
Screen Shot 2021-01-07 at 11 24 11 PM

### CODE REVIEW

File: src/components/HackerActions.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ const HackerActions = () => (
             and organizing the event! This is a blub about the core team 
             and why we love hackbeanpot and organizing the event!
         </p>
-        <a href="#">
+        <a href="/">
```
Comment (cindy1u0) [inline src/components/HackerActions.js]: hm idk why after pulling this from before, it didn't get updated

File: src/assets/fake_data.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
+{
```
Comment (snwessel) [inline src/assets/fake_data.json]: For the sake of consistency, would you mind moving this data file to the `src/data/` directory?

**Review by snwessel — APPROVED** [review]
This looks great! I agree that it will need a bit more work to make it mobile friendly (like we may need to switch away from table elements), but I think this is good to merge and we can iterate from here. I just left one suggestion in a comment on this review 

Thanks for working on this, 😃 and sorry for the delay in reviewing!

File: src/assets/fake_data.json (hunk 2 above)
Comment (cindy1u0) [reply src/assets/fake_data.json]: yep yep!

**Comment by cindy1u0** [comment]
I think I might need to wait for another approval in order to merge this

**Review by daniaalnahdi — APPROVED** [review]
LGTM!

**Pull request merged**